src/django/api/templates/mail/claim_facility_submitted_body.txt

```diff
@@ -1,28 +1,15 @@
 {% block content %}
-Hi,
+Hello,
 
-You're receiving this email because you were listed as the contact for a facility claim request on Open Supply Hub. We'll review the request and be in touch soon. The expected processing time for the initial review of a claim is 10 - 14 days. 
+You are receiving this email because you submitted a request to claim the following production location on Open Supply Hub:
 
-Vendors can refer facilities to claim their profiles, however, the Facility Claim request will only be approved when it is submitted by a facility owner and/or senior management staff at the facility location in question.  If this claim is a referral, please visit our website [https://opensupplyhub.org/auth/register], and register as a user on Open Supply Hub. 
+- Name of Production Location: {{ facility_name }}
+- Address of Production Location: {{ facility_address }}
```

Comment:
There are some inconsistencies in the text/data we show the user in the email between the edited HTML file and this TXT file. Is this expected?
Edited HTML file is in this PR -> https://github.com/opensupplyhub/open-supply-hub/pull/249